packages/web-lib/hooks/useBalances.tsx

```diff
@@ -20,53 +20,75 @@ const		defaultStatus = {
 **************************************************************************/
 function	useBalances(props?: Types.TUseBalancesReq): Types.TUseBalancesRes {
 	const	{networks} = useSettings();
 	const	{address: web3Address, chainID: web3ChainID, isActive, provider} = useWeb3();
-	const	[data, set_data] = useState<{[key: string]: Types.TBalanceData}>({});
+	const	[rawData, set_rawData] = useState<{[key: string]: Types.TBalanceData}>(defaultData);
+	const	[data, set_data] = useState<{[key: string]: Types.TBalanceData}>(defaultData);
 	const	[status, set_status] = useState<Types.TDefaultStatus>(defaultStatus);
 	const	[error, set_error] = useState<Error | undefined>(undefined);
 	const	interval = useRef<NodeJS.Timer>();
-  
+	const	effectDependencies = props?.effectDependencies || [];
+
 	/* 🔵 - Yearn Finance ******************************************************
 	** When this hook is called, it will fetch the informations for the
 	** specified list of tokens. If no props are specified, the default values
 	** will be used.
 	**************************************************************************/
-	const getBalances = useCallback(async (): Promise<{[key: string]: Types.TBalanceData}> => {
+	const getBalances = useCallback(async (): Promise<void> => {
 		if (!isActive || !web3Address || (props?.tokens || []).length === 0) {
-			return {};
+			return;
 		}
 
-		set_status({...defaultStatus, isLoading: true, isFetching: true, isRefetching: defaultStatus.isFetched ? true : false});
+		set_status({
+			...defaultStatus,
+			isLoading: true,
+			isFetching: true,
+			isRefetching: defaultStatus.isFetched ? true : false
+		});
+
 		let		currentProvider = providers.getProvider(props?.chainID || web3ChainID || 1);
 		if (props?.chainID === web3ChainID && provider) {
 			currentProvider = provider as ethers.providers.BaseProvider | ethers.providers.Web3Provider;
 		}
 
-		const	{lensAddress} = networks[props?.chainID || web3ChainID];
-		let		lensContract: Contract | undefined = undefined;
-		if (lensAddress) {
-			lensContract = new Contract(lensAddress, LENS_ABI);
-		}
-
 		const	calls = [];
 		const	ethcallProvider = await providers.newEthCallProvider(currentProvider);
 		for (const element of (props?.tokens || [])) {
 			const	token = element.token as string;
 			const	ownerAddress = (element?.for || web3Address) as string;
-			const	tokenContract = new Contract(token, ERC20_ABI);
-
-			calls.push(...[
-				tokenContract.balanceOf(ownerAddress),
-				tokenContract.decimals(),
-				tokenContract.symbol(),
-				lensContract ? lensContract.getPriceUsdcRecommended(token) : tokenContract.balanceOf(ownerAddress)
-			]);
+			const	isEth = toAddress(token) === ETH_TOKEN_ADDRESS;
+			if (isEth) {
+				const	wETHAddress = WETH_TOKEN_ADDRESS;
+				const	tokenContract = new Contract(wETHAddress, ERC20_ABI);
+				calls.push(...[
+					ethcallProvider.getEthBalance(ownerAddress),
+					tokenContract.decimals(),
+					tokenContract.symbol()
+				]);
```

Comment:
```suggestion
				calls.push(
					ethcallProvider.getEthBalance(ownerAddress),
					tokenContract.decimals(),
					tokenContract.symbol()
				);
```
It's unnecessary to put the elements in an array to just spread them immediately, we can just give multiple args to the push()